README.md

```diff
@@ -146,6 +146,31 @@ BetoCQ takes three steps to address this issue:
     google.com. Note that in China, this test requires an office VPN network or
     installing a VPN app in devices.
 
+*   **Test host.**
+    The test host should be a Linux or Windows machine with the following
+    libraries installed:
+    - python3.11 or later
+      - Check your Python 3 version number:
+
+      ```
+      python3 --version
+      ```
+
+      - If your version is lower than Python 3.11, install the latest version:
+
+      ```
+      sudo apt install python3
+      ```
+      Or install the latest version from [python.org](https://www.python.org/downloads/windows) for Windows.
+    - ADB
+      - If you don't already have the `adb` command-line tool, download and
+        install it from
+        [Android SDK Platform Tools](https://developer.android.com/tools/releases/platform-tools#downloads).
+      - Make sure that the installed binary is in the host's `PATH`, so it can
+        be run directly with `adb` in the command line.
+        - For Windows, follow
+          [these instructions](https://stackoverflow.com/questions/44272416/how-to-add-a-folder-to-path-environment-variable-in-windows-10-with-screensho).
```

Comment:
I think Android has official doc on how to get adb on win right? is this stackoverflow link really the best we have?

Reply:
Removed. The other link already includes the instructions for Windows